jena-base/src/main/java/org/apache/jena/atlas/lib/CollectionUtils.java

```diff
@@ -22,12 +22,7 @@
 
 public class CollectionUtils
 {   
-    public static <T> boolean disjoint(Collection<T> c1, Collection<T> c2)
-    {
-		return c1.stream().noneMatch(c2::contains);
-    }
-    
-    /** Test for same elements, regarless of cardinality */ 
+    /** Test for same elements, regardless of cardinality */ 
     public static <T> boolean sameElts(Collection<T> left, Collection<T> right) {
```

Comment:
Ah, got it-- okay, I will pull out the changes to the `OpVisitor` type and then I think we are good?